app/reducers/rootReducer.ts

```diff
@@ -126,7 +126,8 @@ export const reducer = (state = INITIAL_POPUP_STATE, action: Action) => {
       });
     case actionType.CLEAR_REQUESTS:
       return extendTabRecords(state, action.payload, {
-        requests: []
+        requests: [],
+        requestRecords: {}
```

Comment:
@amit-mb  I think now the requestRecords will get cleared from the redux store.

Reply:
Yes, this is what I was referring to 😄